### PR TITLE
Implement handling of pinot binary columns

### PR DESCRIPTION
**Pull request by svkap**
Test plan

* Have BigDecimal columns in Pinot table (saved as binary). 
* Make a select in Presto for the binary column in Pinot. 
* Test aggregation functions (e.g. sum) and the newly added Pinot UDF: pinot_binary_decimal_to_double. 

```
== RELEASE NOTES ==

General Changes
* Implement handling of Pinot binary columns
* Added Pinot UDF for transforming Pinot binary column to Double. UDF: 'pinot_binary_decimal_to_double'

Hive Changes
* n/a

### CODE REVIEW

File: presto-pinot-toolkit/src/test/java/com/facebook/presto/pinot/TestPinotSegmentPageSource.java [hunk 1]
```diff
@@ -217,6 +219,15 @@ protected static DataTable createDataTableWithAllTypes()
                                     "{ " + generateRandomStringWithLength(RANDOM.nextInt(5)) + " : "
                                         + generateRandomStringWithLength(RANDOM.nextInt(10)) + " }");
                             break;
+                        case BYTES:
+                            try {
+                                dataTableBuilder.setColumn(colId,
+                                        Hex.decodeHex("0DE0B6B3A7640000".toCharArray())); // Hex of BigDecimal.ONE
+                            }
+                            catch (DecoderException e) {
+                                // nothing to do here
```
Comment (xiangfu0) [inline presto-pinot-toolkit/src/test/java/com/facebook/presto/pinot/TestPinotSegmentPageSource.java:228]: just throw an exception here.

**Review by xiangfu0 — COMMENTED** [review]
Overall lgtm, please revert the catalog changes for postgresql.properties.

File: presto-pinot/src/main/java/com/facebook/presto/pinot/udf/PinotFunctions.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            if (returnZeroOnNull) {
+                return 0D;
+            }
+            else {
+                return null;
+            }
+        }
+        return new BigDecimal(new BigInteger(Hex.encodeHexString(input.getBytes()),
+                (int) bigIntegerRadix), (int) scale).doubleValue();
+//        return Decimals.encodeScaledValue(new BigDecimal(new BigInteger(hex, (int) bigIntegerRadix),
```
Comment (tdcmeehan) [inline presto-pinot/src/main/java/com/facebook/presto/pinot/udf/PinotFunctions.java:51]: Please remove the commented out code

**Review by tdcmeehan — CHANGES_REQUESTED** [review]
One nit, otherwise LGTM

**Review by tdcmeehan — APPROVED** [review]

**Pull request merged**